### PR TITLE
Use progressbar python module

### PR DESCRIPTION
**Pull request by schaefi**
As alternative to the no longer developed wget module
we are switching to the progressbar module. This requires
the code to fetch the data to be implemented through the
requests module to allow utilizing the progressbar as
it is designed. A nice after effect is that the progressbar
displayed is more sophisticated than the one before

### CODE REVIEW

File: .virtualenv.requirements.txt [hunk 1]
```diff
@@ -18,4 +18,4 @@ cerberus
 kiwi
 
 # download with progress info
-wget
+progressbar
```
Comment (Conan-Kudo) [inline .virtualenv.requirements.txt:21]: `progressbar2` is the maintained, API-compatible fork of `progressbar`. Use that instead.

File: package/python-kiwi_boxed_plugin-spec-template [hunk 2]
```diff
@@ -73,7 +73,7 @@ Requires:       python%{python3_pkgversion}-docopt
 Requires:       python%{python3_pkgversion}-kiwi >= 9.21.21
 Requires:       python%{python3_pkgversion}-requests
 Requires:       python%{python3_pkgversion}-setuptools
-Requires:       python%{python3_pkgversion}-wget
+Requires:       python%{python3_pkgversion}-progressbar
```
Comment (Conan-Kudo) [inline package/python-kiwi_boxed_plugin-spec-template:76]: `progressbar2` is the maintained, API-compatible fork of `progressbar`. Use that instead.

File: setup.py [hunk 3]
```diff
@@ -29,7 +29,7 @@
         'requests',
         'PyYAML',
         'cerberus',
-        'wget'
+        'progressbar'
```
Comment (Conan-Kudo) [inline setup.py:32]: `progressbar2` is the maintained, API-compatible fork of `progressbar`. Use that instead.

**Review by Conan-Kudo — CHANGES_REQUESTED** [review]

**Comment by Conan-Kudo** [comment]
Cf. https://fedoraproject.org/wiki/Changes/Python-progressbar2_as_default

File: .virtualenv.requirements.txt (hunk 1 above)
Comment (schaefi) [reply .virtualenv.requirements.txt]: good point, tested and done

File: package/python-kiwi_boxed_plugin-spec-template (hunk 2 above)
Comment (schaefi) [reply package/python-kiwi_boxed_plugin-spec-template]: done

File: setup.py (hunk 3 above)
Comment (schaefi) [reply setup.py]: done

**Comment by schaefi** [comment]
going to merge into one if this is acceptable now. Thanks

**Review by Conan-Kudo — APPROVED** [review]

**Pull request merged**